src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -814,6 +832,18 @@ public boolean isRemote()
         return false;
     }
 
+    @Override
```

Comment:
Okey i do not understand your definition logic here:
Hey i do have a research manager cause i was asked to implement it -> But I still refuse to rewrite the surface properly and just leave the methods here?

These two methods have nothing directly to do with the colony -> The colony needs to give access to the manager (as with all other managers) and the manager gives access to these two methods. Not the colony gives access to the managers values....